db/mongodb.js

```diff
@@ -44,7 +44,12 @@ let save = function(reviews, callback) {
 function insertOne(restaurant, callback) {
   Restaurant.create(restaurant, callback);
 }
+
+function findAll(restaurant, callback) {
+  Restaurant.create(restaurant, callback);
```

Comment:
Hey, just curious why the function is called `findAll` but it looks like it's creating a restaurant instead of running a query?

Reply:
good catch, I adjusted this to run a query instead